frontend/src/components/Row.tsx

```diff
@@ -0,0 +1,16 @@
+interface RowProps{
+    name: string;
+    thresholdQuantity: number;
+    costPerUnit: number;
+    shelfLife: string;
+}
+export default function WasteRow({name, thresholdQuantity, costPerUnit, shelfLife}: RowProps) {
```

Comment:
I'm not sure its better to break out the properties of these, or to pass in an object that has all of them. I would tend to prefer the object. That way when we get the response back from the server, we can just pass it down to here without having to break it out.

Reply:
This file has now been replaced entirely by the Row.tsx file. I've also followed your advice and now all properties come in as an object.